x-pack/test/security_solution_endpoint/apps/endpoint/policy_list.ts

```diff
@@ -106,9 +101,19 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await pageObjects.policy.launchAndFindDeleteModal();
         await testSubjects.existOrFail('policyListDeleteModal');
         await pageObjects.common.clickConfirmOnModal();
-        await pageObjects.endpoint.waitForTableToNotHaveData('policyTable');
-        const policyTotal = await testSubjects.getVisibleText('policyTotalCount');
-        expect(policyTotal).to.equal('0 Policies');
+        let emptyPolicyTable;
+        await retry.waitForWithTimeout(
```

Comment:
Why did you need to use `retry.*`? most of the methods in `testSubjects` already use it, so just doing:

```typescript
const emptyPolicyTable = await testSubjects.find('emptyPolicyTable');
```

should work (no?) 😃